lib/moribus/extensions/delegate_associated.rb

```diff
@@ -33,7 +33,7 @@ module DelegateAssociated
       # presented in Customer, the code will result in exception without
       # the following hook:
       def column_for_attribute(name)
-        unless (column = super).nil?
+        unless (column = super).nil? || column.is_a?(ActiveRecord::ConnectionAdapters::NullColumn)
```

Comment:
`ActiveRecord::ConnectionAdapters::NullColumn` is undefined in Rails 4.2.

Reply:
I still forget about lazy/conditional constant evaluation.